packages/react-instantsearch/rollup.config.js

```diff
@@ -11,23 +11,19 @@ const clear = x => x.filter(Boolean);
 const version = process.env.VERSION || 'UNRELEASED';
 const algolia = '© Algolia, inc.';
 const link = 'https://community.algolia.com/react-instantsearch';
-const license = `/*! ReactInstantSearch ${version} | ${algolia} | ${link} */`;
+const createLicence = () =>
+  `/*! ReactInstantSearch ${version} | ${algolia} | ${link} */`;
```

Comment:
`React InstantSearch` should be the actual name (sorry yes it was wrong since the begining)